cirq/experiments/random_quantum_circuit_test.py

```diff
@@ -0,0 +1,233 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cirq
+
+
+def test_random_quantum_circuit():
```

Comment:
This test verifies too little too strictly. Consequently, when someone changes the RQC generation algorithm in future, they'll be forced to re-run it and copy-paste the new circuit for comparison. If in the process an important property of the circuits which we may want to keep invariant changes, we would not be alerted to such unintended consequence.

Instead, we should test for specific properties we want these circuits to have, e.g.:
- each layer contains either only single-qubit gates or only two-qubit gates,
- single- and two-qubit layers alternate,
- first and last layer are single-qubit layers,
- single-qubit gates do not repeat between layers k and k+1,
- two-qubit gate patterns cycle as appropriate.

Reply:
Done.